aptoide-ui/src/main/java/cm/aptoide/pt/aptoide_ui/toolbar/Topbar.kt

```diff
@@ -54,10 +52,9 @@ fun TopBar(
         .fillMaxWidth()
         .weight(1f),
       textAlign = TextAlign.Center,
-      overflow = TextOverflow.Ellipsis,
+      overflow = TextOverflow.Visible,
       maxLines = 1,
       style = AppTheme.typography.medium_M
     )
-    Spacer(Modifier.width(64.dp))
```

Comment:
This causes the topbar title to not be centered. Check topbar in DT settings screen for an example on how to do this